features_api/runtime/sql/functions.sql

```diff
@@ -0,0 +1,3 @@
+CREATE OR REPLACE VIEW pg_temp.eis_fire_snapshot_perimeter_nrt_latest AS
```

Comment:
```suggestion
CREATE OR REPLACE VIEW eis_fire_snapshot_perimeter_nrt_latest AS
```

no need to put `pg_temp` prefix, `tipg` will do it